app.js

```diff
@@ -0,0 +1,8 @@
+const express = require("express");
+const { getTopics } = require('./controllers/controllers')
+const app = express();
+app.use(express.json())
```

Comment:
You will need this later but are we dealing with any request bodies yet?

Reply:
okay tests for errors added, and app.use commented